src/components/validatedInput.jsx

```diff
@@ -16,9 +17,9 @@ const HourInput = React.createClass({
     },
 
     validationStyle () {
```

Comment:
`validationBsStyle` (make bootstrap part explicit, cause this is not a normal style)